type.proto

```diff
@@ -73,6 +73,32 @@ enum ResponseStatus{
     active_disruption = 10;
 }
 
+enum FreeFloatingType {
+    BIKE = 1;
+    SCOOTER = 2;
+    MOTORSCOOTER = 3;
+    STATION = 4;
+    CAR = 5;
+    OTHER = 6;
+}
+
+enum FreeFloatingAttributeType {
+    GEARS = 1;
+    ELECTRIC = 2;
```

Comment:
You have `ELECTRIC` in both `FreeFloatingAttributeType` and `FreeFloatingPropulsionType`
Is it what you want ?

Reply:
Yes. These are two different things as described in https://flow.fluctuo.com/api/Enum/PropulsionType and  https://flow.fluctuo.com/api/Enum/VehicleAttribute

Reply:
:+1: